src/schema/v2/partner/createPartnerContactMutation.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+
+// Check whats required and whats optional
+interface Input {
+  partnerID: string
+  name?: string
+  position?: string
+  canContact?: boolean
+  email?: string
+  emailConfirmation?: string
+  phone?: string
+  locationID?: string
+}
+
+const CreatePartnerContactMutationResponseType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "CreatePartnerContactResponse",
+  fields: () => ({
+    id: { type: new GraphQLNonNull(GraphQLString) },
+    name: { type: GraphQLString },
+    position: { type: GraphQLString },
+    canContact: { type: GraphQLBoolean },
+    email: { type: GraphQLString },
+    phone: { type: GraphQLString },
+    locationID: { type: GraphQLString },
+  }),
+})
+
+export const createPartnerContactMutation = mutationWithClientMutationId<
+  Input,
+  any,
+  ResolverContext
+>({
+  name: "createPartnerContact",
+  description: "Creates a new contact for a partner",
+  inputFields: {
+    partnerID: {
```

Comment:
Currently, this `partnerID` is expecting the string ID (and not a slug)

```
mutation {
  createPartnerContact(input: {
    partnerID: "5f80bfefe8d808000ea212c1"
    name: "Jane Doe"
// yada yada
  })
```

Any objections there?